docs/example-react-router.mdx

```diff
@@ -3,7 +3,159 @@ id: example-react-router
 title: React Router
 ---
 
-This example demonstrates React Router v6. For previous versions see below.
+This example demonstrates React Router v6.4 and above. For previous versions see below.
+
+```jsx
+// App.jsx
+import {
+  createBrowserRouter,
+  Form,
+  Link,
+  Outlet,
+  redirect,
+  RouterProvider,
+  useLoaderData,
+  useLocation,
+} from 'react-router-dom';
+
+// Method to introduce an artificial delay
+export function sleep(n = 500) {
+  return new Promise((r) => setTimeout(r, n));
+}
+
+// Loader to return after a small delay
+export async function homeLoader() {
+  await sleep();
+  return {
+    message: 'home',
+  };
+}
+
+// Action to get user input
+export async function aboutAction({ request }) {
```

Comment:
I think this can also be left out, or there should be a test that verifies its behavior.